sys/cloudformation/stack.yaml

```diff
@@ -165,370 +230,343 @@ Resources:
           IpProtocol: tcp
           FromPort: 443
           ToPort: 443
-      VpcId: !Ref 'VpcId'
+      VpcId: !Ref VpcId
       Tags:
         - Key: env
-          Value: badge-poser
+          Value: !Ref ServiceName
   sgredis:
     Type: AWS::EC2::SecurityGroup
     Properties:
-      GroupDescription: badge-poser-redis
-      GroupName: badge-poser-redis
+      GroupDescription: !Sub '${ServiceName}-redis'
+      GroupName: !Sub '${ServiceName}-redis'
       SecurityGroupEgress:
         - CidrIp: 0.0.0.0/0
-          IpProtocol: -1
+          IpProtocol: '-1'
           FromPort: 0
           ToPort: 65535
       SecurityGroupIngress:
         - CidrIp: 0.0.0.0/0
           IpProtocol: tcp
           FromPort: 6379
           ToPort: 6379
-      VpcId: !Ref 'VpcId'
+      VpcId: !Ref VpcId
       Tags:
         - Key: env
-          Value: badge-poser
+          Value: !Ref ServiceName
 
   # CLOUDWATCH LOGS
   cloudwatchloggroup:
     Type: AWS::Logs::LogGroup
     Properties:
-      LogGroupName:
-        Fn::Sub:
-          - "${ServiceName}-logs"
+      LogGroupName: !Sub '${ServiceName}-logs'
+      RetentionInDays: 30
 
   # ELB
   elb:
     Type: AWS::ElasticLoadBalancingV2::LoadBalancer
-    DependsOn:
```

Comment:
is this useless? just to know more about it ;)

Reply:
Yes, as the dependence is already stated by the `!Ref` in "SecurityGroups" property